packages/client/lib/net/protocol/lesprotocol.ts

```diff
@@ -174,10 +174,10 @@ export class LesProtocol extends Protocol {
     }
 
     return {
-      networkId: this.chain.networkId.toNumber(),
+      networkId: this.chain.networkId.toArrayLike(Buffer),
       headTd: this.chain.headers.td.toArrayLike(Buffer),
       headHash: this.chain.headers.latest?.hash(),
-      headNum: this.chain.headers.latest?.number,
+      headNum: this.chain.headers.latest?.number.toArrayLike(Buffer),
```

Comment:
I believe encoding this BN as a Buffer is more correct, according to the [les protocol messages spec](https://github.com/ethereum/devp2p/blob/master/caps/les.md#status-0x00).